packages/TorneloScoresheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -62,6 +68,28 @@ const Toolbar: React.FC = () => {
   const currentColour = colourForMode[appModeState.mode];
   const arbiterModeVisibility = arbiterModeDisplay[appModeState.mode];
   const currentTextColour = textColour(currentColour);
+
+  const graphicalRecordingState = useGraphicalRecordingState();
+  const goToArbiterGameFromGraphicalRecording =
+    graphicalRecordingState?.[1]?.goToArbiterGameMode;
+
+  const tablePairingState = useTablePairingState();
+  const goToArbiterGameFromTablePairing =
+    tablePairingState?.[1]?.goToArbiterGameMode;
+
+  const handleVerify = () => {
```

Comment:
even if we are not handling the checking logic yet, maybe this function should take the pin value as an input and maybe console log it or something witha  todo comment saying we need to implement the check